hub/powertoys/cmd-not-found.md

```diff
@@ -0,0 +1,37 @@
+---
+title: PowerToys Command Not Found for Windows
+description: A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
+ms.date: 02/01/2024
+ms.topic: article
+no-loc: [PowerToys, Windows, Command Not Found, Win]
+---
+
+# Command Not Found utility
+
+A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
```

Comment:
```suggestion
A PowerShell 7 module that detects command-line errors and suggests a relevant WinGet package to install, if available.
```

This is just a small suggestion but I'm not sure about 'thrown' as a verb to describe errors. I know it's used for [exceptions in these docs](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/exception-throwing) but I can't see it in many other Microsoft docs.